### PR TITLE
Issue/80 User not get proper token roles

### LINKED ISSUE #80: User's first sign in doesn't get proper Auth token roles
# Problem
First login doesn't grant proper access to CMS.

# Breakdown
Tanam is using custom claims in the auth object to enforce role based authorization. But the roles are being set in an "on create account" cloud function so the user will already have received the auth token, without claims, before the claims are being set. That effectively makes the user unable to get the access that he/she has been granted in parallel on server side.

# Workaround
Currently, the workaround is to log out and login again immediately after creating an account.

# Proposal
This should be solved by refreshing token in the CMS frontend app.

### PR DESCRIPTION
**Pull request by detautama**
Fixes #80 

This PR make refresh auth-tokens upon updates when changes detected on user

### CODE REVIEW

File: angular/projects/admin/src/app/services/app-auth.service.ts [hunk 1]
```diff
@@ -24,4 +24,21 @@ export class AppAuthService {
   logOut(): Promise<void> {
     return this.fireAuth.auth.signOut();
   }
+
+  async reloadUserService() {
```
Comment (DennisAlund) [inline angular/projects/admin/src/app/services/app-auth.service.ts]: The function should just be called `reloadUser` the class itself is the "service"

File: angular/projects/admin/src/app/admin/navigation/profile-widget/profile-widget.component.ts [hunk 2]
```diff
@@ -22,4 +23,11 @@ export class ProfileWidgetComponent implements OnInit {
     this.appAuthService.logOut();
   }
 
+  private async _reloaduser() {
+    const payload = await this.appAuthService.reloadUserService();
```
Comment (DennisAlund) [inline angular/projects/admin/src/app/admin/navigation/profile-widget/profile-widget.component.ts]: Rename `reloadUserService` -> `reloadUser`

Comment (detautama) [reply angular/projects/admin/src/app/admin/navigation/profile-widget/profile-widget.component.ts]: Resolve on this commit https://github.com/oddbit/tanam/pull/195/commits/87684dd3dcaa2d579fc888e3fc60e9e159a3dc2e

File: angular/projects/admin/src/app/services/app-auth.service.ts (hunk 1 above)
Comment (detautama) [reply angular/projects/admin/src/app/services/app-auth.service.ts]: Resolve on this commit https://github.com/oddbit/tanam/pull/195/commits/87684dd3dcaa2d579fc888e3fc60e9e159a3dc2e

**Pull request merged**